lib/fluent/plugin/in_cat_sweep.rb

```diff
@@ -1,3 +1,4 @@
+require 'fluent/input'
```

Comment:
This is needed to work with Fluentd v0.14 compatible layer.
And this is also needed for Fluentd v0.12.3x or later.